integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -1642,7 +1638,9 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
     var rows = sql("SHOW STREAMS").collect()
     assertResult(0)(rows.length)
 
-    val csvDataDir = new File("target/csvdata").getCanonicalPath
+    val csvDataDir = integrationPath + "/target/csvdata"
```

Comment:
it should be integrationPath + "/spark2/target/csvdatanew

Reply:
fixed